Ghidra/Processors/PowerPC/data/languages/ppc_instructions.sinc

```diff
@@ -1930,8 +1930,9 @@
 	zero:4=0;
 	fD=fC;
 	cr1flags();
-	if (tmpfA f> int2float(zero)) goto inst_next;
+	if (tmpfA f>= int2float(zero)) goto inst_next;
```

Comment:
Rather than duplicate `cr1flags` it would be better to refactor this as:
```
	local tmpfA = fA;
	local tmpfB = fB;
	fD=fC;
	if (tmpfA f> int2float(zero)) goto <end>;
	fD=tmpfB;
<end>
	cr1flags();
```

Reply:
Ah, makes sense (I presume you meant to keep `f>=`, not revert back to `f>` though). I'll apply that change then; should I squash it onto the initial commit or can I just leave it as a separate one?

Reply:
sorry, yes, that was a copy-paste error

> Ah, makes sense (I presume you meant to keep `f>=`, not revert back to `f>` though). I'll apply that change then; should I squash it onto the initial commit or can I just leave it as a separate one?

Sorry, yes, that was a copy paste error. Squashed if you don't mind.